### ISSUE TITLE
Fuzz test error

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/circuit-construction-kit-common#309]
```
circuit-construction-kit-dc : fuzz : require.js : run
Uncaught NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at AccessibleInstance.dispose (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/accessibility/AccessibleInstance.js?bust=1494431682864:312:53)
    at AccessibleInstance.removeSubtree (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/accessibility/AccessibleInstance.js?bust=1494431682864:159:25)
    at Display.removeAccessibleTrail (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/display/Display.js?bust=1494431682864:653:47)
    at HBox.onAccessibleRemoveChild (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/nodes/Node.js?bust=1494431682864:1726:31)
    at HBox.removeChildWithIndex (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/nodes/Node.js?bust=1494431682864:672:14)
    at HBox.removeChild (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/nodes/Node.js?bust=1494431682864:629:12)
    at HBox.setChildren (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/nodes/Node.js?bust=1494431682864:717:16)
    at HBox.setChildren (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/nodes/LayoutBox.js?bust=1494431682864:246:34)
    at HBox.removeAllChildren (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/scenery/js/nodes/Node.js?bust=1494431682864:698:12)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1494430401061/joist/js/HomeScreenView.js?bust=1494431682864:134:14)
Approximately 5/10/2017, 9:33:21 AM
```

**Comment by jessegreenberg** [comment]
This can happen by opening the About Dialog and then switching screens.

**Comment by jessegreenberg** [comment]
Ah, after closing the Dialog there are some things in the DOM that shouldn't be there.
![capture](https://cloud.githubusercontent.com/assets/6396244/25914274/f52d62c8-357a-11e7-8b50-33b870e98aa4.PNG)

That looks like the accessible content for the circuit elements in the screen icons.

**Comment by jessegreenberg** [comment]
The icons are using ```this.accessibleContent = null``` to remove content from the DOM, which is ignored when the Dialog is hidden again.  This is a Scenery problem, and I will create an issue there.

**Comment by jessegreenberg** [comment]
Fixed with the above commit and https://github.com/phetsims/scenery/issues/614, fuzz test isn't hitting anything.
